ext/storage_sources/gcp_store/gcp_connection.h

```diff
@@ -45,8 +45,9 @@ class gcp_connection {
     ~gcp_connection() = default;
 
     private:
-    const google::cloud::storage::v2_5_0::Client _gcp_client;
+    google::cloud::storage::v2_5_0::Client _gcp_client;
     const std::string _bucket_name;
+    const std::string _prefix;
```

Comment:
name the prefix the same as azure as well!

Reply:
yup, renamed here: [bab1c68](https://github.com/wiredtiger/wiredtiger/pull/8717/commits/bab1c686c89274e4a4d53f944ec1dab0e87f637c)